src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon, IconKey } from "src/components/Icon";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconButton } from "../IconButton";
+import { useToastContext } from "./ToastContext";
+
+export type ToastTypes = "error" | "warning" | "success" | "info" | "alert";
```

Comment:
**nit (non-blocking):** Would prefer to move `types` to the bottom of the file. This is per our component style guide: https://homebound.atlassian.net/wiki/spaces/ENG/pages/339574863/Component+Style+Guide#Component-Structure